packages/tiny-router/src/__tests__/index.test.ts

```diff
@@ -13,10 +13,13 @@ let get: jest.Mock;
 
 const createStore = (config) => frontity.createStore<Packages>(config);
 
+const spiedPushState = jest.spyOn(window.history, "pushState");
+const spiedReplaceState = jest.spyOn(window.history, "replaceState");
+
 beforeEach(() => {
   normalize = jest.fn().mockImplementation((link) => {
-    const { pathname, hash, search } = new URL(link, "https://dummy.com");
-    return pathname + hash + search;
+    const { pathname, search, hash } = new URL(link, "https://dummy.com");
+    return pathname + search + hash;
```

Comment:
I thought this code was wrong only in this branch but it's in `dev` too! I've added a changeset for these changes.

Reply:
Great!